### PR TITLE
Ru help encoding

### PR DESCRIPTION
**Pull request by pm-kan**

### CODE REVIEW

File: help/pwsafeRU/html/credits.html [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
 <html>
 <head>
-    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
-<title>Благодарности</title><!-- pwsafe version: 3.12--><!-- Modified: 10/12/2007 --><!--Popup control -->
+    <meta http-equiv="content-type" content="text/html; charset=Windows-1251">
+<title>�������������</title><!-- pwsafe version: 3.12--><!-- Modified: 10/12/2007 --><!--Popup control -->
```
Comment (dskrvk) [inline help/pwsafeRU/html/credits.html]: This file renders weird unlike the others

File: help/pwsafeRU/html/help_menu.html [hunk 2]
```diff
@@ -1,8 +1,8 @@
 <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
 <html>
 <head>
-    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
-<title>Меню «Помощь»</title><!-- pwsafe version: 2.11--><!-- Modified: 15/06/2005 --><!--Popup control -->
+    <meta http-equiv="content-type" content="text/html; charset=Windows-1251">
+<title>���� �������</title><!-- pwsafe version: 2.11--><!-- Modified: 15/06/2005 --><!--Popup control -->
```
Comment (dskrvk) [inline help/pwsafeRU/html/help_menu.html]: Another one

File: help/pwsafeRU/html/menu_toolbar.html [hunk 3]
```diff
@@ -1,8 +1,8 @@
 <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
 <html>
 <head>
-    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
-<title>Панель инструментов</title><!-- pwsafe version: 2.11--><!-- Modified: 15/06/2005 --><!--Popup control -->
+    <meta http-equiv="content-type" content="text/html; charset=Windows-1251">
+<title>������ ������������</title><!-- pwsafe version: 2.11--><!-- Modified: 15/06/2005 --><!--Popup control -->
```
Comment (dskrvk) [inline help/pwsafeRU/html/menu_toolbar.html]: Another one

File: help/pwsafeRU/html/credits.html (hunk 1 above)
Comment (pm-kan) [reply help/pwsafeRU/html/credits.html]: It should be fine. It seems, that Github can't detect codepage changes in these files (it was surprise to me, that in other files it recoded diff lines for new revision in preview). You could take a look at raw file content at my repo, for example https://raw.githubusercontent.com/pm-kan/pwsafe/ru-help-encoding/help/pwsafeRU/html/credits.html (don't forget to change browser's encoding to Windows-1251)

Comment (dskrvk) [reply help/pwsafeRU/html/credits.html]: Yeah, that worked. Not sure why the others render fine by default.

**Pull request merged**